app_version.hpp.in

```diff
@@ -0,0 +1,19 @@
+package version
```

Comment:
This package will not be built without cmake and no fallback template file provided. So package cannot be built with plain `go build`.